### PR TITLE
Master

### PR DESCRIPTION
**Pull request by nbhaan**
First prototype

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -47,41 +57,99 @@ function startNewSession() {
...
+		document.getElementById("digitalTime").style.backgroundImage = "url('css/afternoon.png')";
+		return;
+	} else if (hours < 24){
+		document.getElementById("digitalTime").style.backgroundImage = "url('css/evening.png')";
+		return;
+	}
 }
 
+function printDigitalTime() {
+	'use strict';
```
Comment (mircealungu) [inline js/main.js]: please verify if it's sufficient to use 'use strict' once per file.

File: js/main.js [hunk 2]
```diff
@@ -70,14 +70,18 @@ window.onload = function() {
...
-    document.getElementById("correctButton").addEventListener("click", function(){
+    document.getElementById("nextButton").addEventListener("click", function(){
         wordNumber++;
         if (wordNumber > numberOfWords) {
         	wordNumber = 0;
         }
     	ctx.clearRect(0,0, 380,380);
     	printOnScreen(words[wordNumber], 180, 50);
-        printOnScreen(translations[wordNumber], 180, 100);
+
```
Comment (mircealungu) [inline js/main.js]: no magic numbers in the code. 
extract these into constants.

File: js/app/time.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+		create: function() {
+			date = getTizenDateTime();
+			year = date.getFullYear();
+			day = date.getDate();
+			month = date.getMonth() + 1;
+			hours = date.getHours();
+			minutes = date.getMinutes();
+			seconds = date.getSeconds();
+
+			if (month < 10) {
```
Comment (digeo) [inline js/app/time.js]: Do not repeat yourself!
create one function:
`function xyz(value) {
   if (value < 10)
       return "0" + value;
return value
}
`

File: css/style.css [hunk 4]
```diff
@@ -1,37 +1,50 @@
...
 }
 
-.footerButton{
-	height: 80px;
-	width: 160px;
+.footerButtonLeft{
+	width: 170px;
+	height: 120px;
+	border-radius: 20px;
+	background: url('reveal.png');
```
Comment (digeo) [inline css/style.css]: I will write it here.
Move all the images to a new directory (i.e. assets)
They are not css

File: js/main.js [hunk 5]
```diff
@@ -63,6 +63,9 @@ function printOnScreen(string, posx, posy, size) {
 	ctxWords.fillStyle = "#FFFFFF";
```
Comment (digeo) [inline js/main.js]: create constant: FILL_STYLE_COLOR = "#FFFFFF";

File: js/main.js [hunk 6]
```diff
@@ -63,6 +63,9 @@ function printOnScreen(string, posx, posy, size) {
...
 	ctxWords.textAlign = "center";
```
Comment (digeo) [inline js/main.js]: create constant: CTX_WORDS_TEXT_ALIGN = "center";

File: js/main.js [hunk 7]
```diff
@@ -63,6 +63,9 @@ function printOnScreen(string, posx, posy, size) {
...
 	ctxWords.fillText(string, posx, posy);
+	// print length of word in pixels
```
Comment (digeo) [inline js/main.js]: remove comments of your code

File: js/main.js [hunk 8]
```diff
@@ -153,6 +153,8 @@ window.onload = function() {
     });
     
     document.getElementById("revealButton").addEventListener("click", function(){
+    	ctxWords.clearRect(0,0, 380,380);
```
Comment (digeo) [inline js/main.js]: extract all the values to constants

File: .sign/.manifest.tmp [hunk 9]
```diff
@@ -1,137 +1,6 @@
-.DS_Store__DEL__ZZr7nHdvp979KE+Mau6S+zA8+LL3vz/0NlqZKML6klY=
```
Comment (digeo) [inline .sign/.manifest.tmp]: please define a good .gitignore!

File: js/main.js [hunk 10]
```diff
@@ -133,42 +164,50 @@ function printDigitalTime() {
...
+	document.addEventListener('tizenhwkey', function(e) {
         if (e.keyName === "back") {
             try {
                 tizen.application.getCurrentApplication().exit();
             } catch (ignore) {}
         }
     });
+}
+
+function printFirstWord(){
```
Comment (digeo) [inline js/main.js]: Maybe you should refactor this function to:
`function printWord(word){
    printOnScreen(word SCREEN_MIDDLE, WORD_POSY, MAX_TEXTSIZE);
}`

File: js/main.js [hunk 11]
```diff
@@ -151,28 +146,27 @@ function printDigitalTime() {
 	ctxTime.font = MAX_TEXTSIZE + FONT;
 	ctxTime.fillStyle = BLACK;
 	ctxTime.textAlign = CENTER;
-    ctxTime.fillText(hours + ":" + minutes, SCREEN_MIDDLE, DIGITAL_TIME_POSY);
+    ctxTime.fillText(hours + ":" + minutes, SCREEN_WIDTH/2, DIGITAL_TIME_POSY);
```
Comment (digeo) [inline js/main.js]: create one new constant: `var XYZ = SCREEN_WIDTH/2`

File: js/main.js [hunk 12]
```diff
@@ -28,17 +28,12 @@ var USERNAME = 'session/i@mir.lu';
 //definitions for coordinates
 var SCREEN_WIDTH = 360;
```
Comment (digeo) [inline js/main.js]: Why do you have hard-coded values for the width of the screen?
Do all the smartwatches have 360 width?

**Pull request merged**